<html>
<head>
<title>TestNG:  GroupDependence</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>GroupDependence</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>5/1/1</td>
</tr><tr>
<td>Started on:</td><td>Thu Aug 16 15:20:27 CST 2018</td>
</tr>
<tr><td>Total time:</td><td>0 seconds (28 ms)</td>
</tr><tr>
<td>Included groups:</td><td>init func </td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='trainTestNG.TestFunc.testMethod1()'><b>testMethod1</b><br>Test class: trainTestNG.TestFunc ()</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at trainTestNG.TestFunc.testMethod1(TestFunc.java:31)
... Removed 27 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1025799482", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1025799482'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at trainTestNG.TestFunc.testMethod1(TestFunc.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1246)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1171)
	at org.testng.TestNG.run(TestNG.java:1066)
	at org.testng.TestNG.privateMain(TestNG.java:1387)
	at org.testng.TestNG.main(TestNG.java:1356)
</pre></div></td>
<td>0</td>
<td>trainTestNG.TestFunc@233c0b17</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='trainTestNG.TestBase.checkMethod()'><b>checkMethod</b><br>Test class: trainTestNG.TestBase ()</td>
<td></td>
<td>0</td>
<td>trainTestNG.TestBase@61443d8f</td></tr>
<tr>
<td title='trainTestNG.TestBase.initial()'><b>initial</b><br>Test class: trainTestNG.TestBase ()</td>
<td></td>
<td>0</td>
<td>trainTestNG.TestBase@61443d8f</td></tr>
<tr>
<td title='trainTestNG.TestBase.setup()'><b>setup</b><br>Test class: trainTestNG.TestBase ()</td>
<td></td>
<td>0</td>
<td>trainTestNG.TestBase@61443d8f</td></tr>
<tr>
<td title='trainTestNG.TestFunc.testMethod2()'><b>testMethod2</b><br>Test class: trainTestNG.TestFunc ()</td>
<td></td>
<td>0</td>
<td>trainTestNG.TestFunc@233c0b17</td></tr>
<tr>
<td title='trainTestNG.TestFunc.testMethod3()'><b>testMethod3</b><br>Test class: trainTestNG.TestFunc ()</td>
<td></td>
<td>0</td>
<td>trainTestNG.TestFunc@233c0b17</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='trainTestNG.TestFunc.testMethod4()'><b>testMethod4</b><br>Test class: trainTestNG.TestFunc ()</td>
<td><div><pre>java.lang.Throwable: Method TestFunc.testMethod4()[pri:0, instance:trainTestNG.TestFunc@233c0b17] depends on not successfully finished methods
... Removed 16 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1908316405", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1908316405'><pre>java.lang.Throwable: Method TestFunc.testMethod4()[pri:0, instance:trainTestNG.TestFunc@233c0b17] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1246)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1171)
	at org.testng.TestNG.run(TestNG.java:1066)
	at org.testng.TestNG.privateMain(TestNG.java:1387)
	at org.testng.TestNG.main(TestNG.java:1356)
</pre></div></td>
<td>0</td>
<td>null</td></tr>
</table><p>
</body>
</html>